index.html

```diff
@@ -4,6 +4,10 @@
 <head>
   <meta charset="UTF-8" />
   <link rel="icon" type="image/svg+xml" href="/skier.svg" />
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet"
+    integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+  <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js"
+    integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM" crossorigin="anonymous"></script>
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
   <title>Apres Ski</title>
 </head>
```

Comment:
Hey @MattWalterTX ! When you get a chance, could you make a commit and rearrange the order in the HTML to match the screenshot? I believe the order of links and scripts can affect how things load in, so I rearranged to match how the Bootstrap template has it: 
![Screen Shot 2023-02-08 at 6 00 48 PM](https://user-images.githubusercontent.com/108706408/217679498-9fe72576-d897-4647-b21f-efca177a3514.png)
 order.

Reply:
No Problem! I slid the script down to just under the root (also tabbed in the button group components on Landing)

Reply:
Thank you!